staging/src/k8s.io/code-generator/cmd/validation-gen/output_tests/pointers/zz_generated.validations.go

```diff
@@ -66,11 +64,6 @@ func Validate_T1(in *T1, fldPath *field.Path) (errs field.ErrorList) {
 		errs = append(errs, validate.FixedResult(fldPath.Child("pf"), *in.PF, true, "field T1.PF")...)
 	}
 
-	// PT2
-	if in.PT2 != nil {
```

Comment:
This is removing validation for PT2 in this test-case, but ...don't we WANT to validate PT2?

Reply:
Ooh, my pruners bugged.  I can drop it from this PR.  I just wanted to filter generated code down, but I think it needs more attention.  I'll drop these commits and stash this work for now.